components/BottomMenu.js

```diff
@@ -27,24 +30,48 @@ function BottomMenu () {
         setSelectedBuilding(name);
     };
 
+    const searchItemSelected = async () => {
```

Comment:
Can we call async function getters? So in this case, `getDestination`

Reply:
Yes, the name could be refactored if we consider them getters, nice catch

Reply:
I'll change it :)